README.md

```diff
@@ -36,6 +36,7 @@ Run the top-level [build script](build.sh) with the `--help` parameter for help.
 | [libpng]        | 1.6.52     | [libpng License version 2]                                   |
 | [librsvg]       | 2.61.3     | LGPLv3                                                       |
 | [libtiff]       | 4.7.1      | [libtiff License] (BSD-like)                                 |
+| [libultrahdr]   | 1.4.0      | Apache-2.0 License                                           |
```

Comment:
Apache-2.0 isn't a copyleft license, could that introduce any issues for us? I recall that Highway was dual-licensed for similar reasons, to ensure compatibility with NumPy's BSD-3 license (see e.g. the removed licensing paragraph in commit https://github.com/numpy/numpy/commit/f09ab71f7c52e182e4f6c947800a9d30d80cdc2a).